monitoring/statuscake/resources.tf

```diff
@@ -1,8 +1,8 @@
 resource "statuscake_uptime_check" "main" {
-  for_each = toset(var.uptime_urls)
+  for_each = var.statuscake_alerts
```

Comment:
this should remain toset(var.uptime_urls) otherwise we break all the existing alerts